packages/form-builder/src/components/FormBuilder/tests/data/questionForm.ts

```diff
@@ -239,11 +239,11 @@ export const questionForm: Form = {
 				title: 'Question choix autre',
 				description: 'Informations supplémentaires',
 				multiple: true,
```

Comment:
```suggestion
				multiple: true,
				type: 'select',
```
Pas besoin de bouger le select 😛